src/commands/dev/index.ts

```diff
@@ -367,6 +169,11 @@ export default class Dev extends BaseCommand {
       sensitive: false,
       default: false,
     }),
+    'wait-timeout': Flags.string({
+      description: '[default: 10m] Time to wait for services to be ready/healthy before detaching.',
```

Comment:
It looks like oclif adds the default text automatically, it shows up twice:

![image](https://user-images.githubusercontent.com/1406391/230974706-51e36548-546e-441e-998e-7a7470b05a39.png)

Reply:
Good find. Found a couple other places where it was duplicated and fixed.